### PR TITLE
add better logging around service failures and spawning

### PR DESCRIPTION
**Pull request by mwrock**
This can be helpful when debugging service crashes.

Signed-off-by: mwrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/launcher/src/sys/unix/service.rs [hunk 1]
```diff
@@ -115,6 +115,7 @@ impl Process {
 }
 
 pub fn run(msg: protocol::Spawn) -> Result<Service> {
+    debug!("spawning {}", msg.get_binary());
```
Comment (christophermaier) [inline components/launcher/src/sys/unix/service.rs]: Might be nice to have a corresponding line over in the Windows implementation, too.

**Comment by mwrock** [comment]
Oh right of course! Will add that.

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/276181082).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/276181082).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**